docs/_substitutions.rst

```diff
@@ -0,0 +1,40 @@
+.. |bibliography| replace:: :ref:`bibliography`\
+.. |glossary| replace:: :ref:`glossary`\
+.. |inf| replace:: `~numpy.inf`
+.. |minpython| replace:: 3.8
+.. |nan| replace:: `~numpy.nan`
+.. |ndarray| replace:: :class:`~numpy.ndarray`
+.. |Quantity| replace:: :class:`~astropy.units.Quantity`
+.. |TimeDelta| replace:: :class:`~astropy.time.TimeDelta`
+.. |Time| replace:: :class:`~astropy.time.Time`
```

Comment:
This file includes some reStructuredText substitutions that we can use throughout our docs, like...
```
This function accepts a |Quantity| with units of mass.  If an |ndarray| is provided, SI units will be assumed.
```